app/models/call.rb

```diff
@@ -27,6 +27,18 @@ class Call
                 track_destroy: true
   mongoid_userstamp user_model: 'User'
 
+  after_save :log_event, if: -> { status_changed? }
+
+  def log_event
```

Comment:
@tedPen I'm interested in your thoughts -- we'll probably have to use this code in patient too, and there's totally a scenario where our users ask for more events. What do you think about throwing `log_event` into its own concern and having it take args that populate the object fields? I'm spitballing here, super open to other stuff, would love to know what you think.